src/MetadataEntry.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of libSavitar
+ *
+ * Copyright (C) 2021 Ultimaker B.V. <j.vankessel@ultimaker.com>
```

Comment:
"In the name of J. van Kessel, I create thee, file!"

Well I presume Jaime's OK with this, it is building on his PR after all  :-p

Reply:
I took this from the rest of the files in libSavitar. I considered putting my own address in there but to be consistent I kept the e-mail address the same. Normally I'd just write the 2 lines we have on CuraEngine source files.